app/services/api/webflow-api.ts

```diff
@@ -23,40 +33,49 @@ const useWebflowAPI = <T>(key: string, collectionId: string, enabled = true) =>
   })
 
 export const useSponsors = () => {
-  return useWebflowAPI<CustomSponsorProps>(SPONSORS.key, SPONSORS.collectionId)
+  return useWebflowAPI<RawSponsor>(SPONSORS.key, SPONSORS.collectionId)
 }
 
 export const useSpeakers = () => {
-  const { data, ...rest } = useWebflowAPI<CustomSpeakerProps>(SPEAKERS.key, SPEAKERS.collectionId)
-  return { data: cleanedSpeakers(data), ...rest }
+  return useWebflowAPI<RawSpeaker>(SPEAKERS.key, SPEAKERS.collectionId)
 }
 
 export const useSpeakerNames = () => {
-  const { data, ...rest } = useWebflowAPI<CustomSpeakerNamesProps>(
-    SPEAKER_NAMES.key,
-    SPEAKER_NAMES.collectionId,
-  )
-  return { data, ...rest }
+  return useWebflowAPI<RawSpeakerName>(SPEAKER_NAMES.key, SPEAKER_NAMES.collectionId)
 }
 
 export const useWorkshops = () => {
-  const { data: speakersData, isLoading } = useSpeakers()
-  const { data: workshopsData, ...rest } = useWebflowAPI<CustomWorkshopProps>(
-    WORKSHOPS.key,
-    WORKSHOPS.collectionId,
-    !isLoading && !!speakersData,
-  )
-  return { data: cleanedWorkshops(workshopsData, cleanedSpeakers(speakersData)), ...rest }
+  return useWebflowAPI<RawWorkshop>(WORKSHOPS.key, WORKSHOPS.collectionId)
+}
+
+export const useRecurringEvents = () => {
+  return useWebflowAPI<RawRecurringEvents>(RECURRING_EVENTS.key, RECURRING_EVENTS.collectionId)
+}
+
+export const useTalks = () => {
+  return useWebflowAPI<RawTalk>(TALKS.key, TALKS.collectionId)
 }
 
-export const useSchedule = () => {
-  const { data: speakersData, isLoading } = useSpeakers()
-  const { data: schedulesData, ...rest } = useWebflowAPI<CustomScheduleProps>(
+export const useScheduledEvents = () => {
+  const { data: speakers, isLoading } = useSpeakers()
+  const { data: workshops } = useWorkshops()
+  const { data: recurringEvents } = useRecurringEvents()
+  const { data: talks } = useTalks()
+  const { data: scheduledEvents, ...rest } = useWebflowAPI<RawScheduledEvent>(
     SCHEDULE.key,
     SCHEDULE.collectionId,
-    !isLoading && !!speakersData,
+    !isLoading && !!speakers,
```

Comment:
Is this hook only dependent upon speakers being loaded? Or does having talks/workshops loaded impact it as well?

Reply:
Honestly, that was just how it was being done before. I'm not sure if I understand the reasoning. Seems like we wouldn't necessarily want automatic refetching. Maybe we could just set them all to false?

From the TS docs:
`Set this to false to disable automatic refetching when the query mounts or changes query keys. To refetch the query, use the refetch method returned from the useQuery instance. Defaults to true.`